src/commands/notify.yml

```diff
@@ -85,6 +86,64 @@ steps:
       name: Slack Notification
       shell: /bin/bash
       command: |
+        # set up attachments json
+        attachments_json='[{
+        	"fallback": "<< parameters.message >> - $CIRCLE_BUILD_URL",
+            "text": "<< parameters.message >> $SLACK_MENTIONS",
+            "author_name": "<< parameters.author_name >>",
+            "author_link":"<< parameters.author_link >>",
+            "title": "<< parameters.title >>",
+            "title_link": "<< parameters.title_link >>",
+            "footer": "<< parameters.footer >>",
+            "ts": "<< parameters.ts >>",
+            "fields": [],
+            "actions": [],
+            "color": "<< parameters.color >>"
+        }]'
+        <<# parameters.include_project_field >>
+        attachments_json=$(echo $attachments_json | jq '.[].fields[.[].fields | length] |= . +
+        {
+        	"title": "Project",
+        	"value": "$CIRCLE_PROJECT_REPONAME",
+        	"short": true
+        }')
+        <</ parameters.include_project_field >>
+        <<# parameters.include_job_number_field >>
+        attachments_json=$(echo $attachments_json | jq '.[].fields[.[].fields | length] |= . +
+        {
+        	"title": "Job Number",
+        	"value": "$CIRCLE_BUILD_NUM",
+        	"short": true
+        }')
+        <</ parameters.include_job_number_field >>
+        <<# parameters.include_visit_job_action >>
+        attachments_json=$(echo $attachments_json | jq '.[].actions[.[].actions | length] |= . +
+        {
+        	"type": "button",
+        	"text": "Visit Job",
+        	"url": "$CIRCLE_BUILD_URL"
+        }')
+        <</ parameters.include_visit_job_action >>
+        <<# parameters.dashboard_link >>
+        attachments_json=$(echo $attachments_json | jq '.[].actions[.[].actions | length] |= . +
+        {
+        	"type": "button",
+        	"text": "Dashboard",
+        	"url": "<< parameters.dashboard_link >>"
+        }')
+        <</ parameters.dashboard_link >>
+        <<# parameters.lighthouse_button_links >>
+        for link in << parameters.lighthouse_ph_results_link >>; do
```

Comment:
more lighthouse refs. These should be generic enough that any report can come in here